test/hga/HGA_Regression.ipynb

```diff
@@ -247,22 +237,16 @@
     "    avnir_output_three = make_request_and_download_result(harmony_client, avnir_request_three,\n",
     "                                                          avnir_directory_three)\n",
     "\n",
-    "    avnir_variable_base_three = 'ALAV2A239652240-OORIRFU_subsetted'\n",
+    "    avnir_variable_base_three = 'ALAV2A239652240-OORIRFU'\n",
     "\n",
-    "    avnir_variables_three = {f'{avnir_variable_base_three}__Band1__colored_band_1',\n",
-    "                             f'{avnir_variable_base_three}__Band1__colored_band_2',\n",
-    "                             f'{avnir_variable_base_three}__Band1__colored_band_3',\n",
-    "                             f'{avnir_variable_base_three}__Band1__colored_band_4',\n",
-    "                             f'{avnir_variable_base_three}__Band2__colored_band_1',\n",
-    "                             f'{avnir_variable_base_three}__Band2__colored_band_2',\n",
-    "                             f'{avnir_variable_base_three}__Band2__colored_band_3',\n",
-    "                             f'{avnir_variable_base_three}__Band2__colored_band_4'}\n",
+    "    avnir_variables_three = {f'IMG-01-{avnir_variable_base_three}_002_band_1',\n",
+    "                             f'IMG-02-{avnir_variable_base_three}_002_band_1'}\n",
```

Comment:
The only weird thing I saw was this granule has `002` in the variables where all the other tests/granules have `000`